FlaskSite/api/v1/topic.py

```diff
@@ -5,22 +5,26 @@
 bp = Blueprint('topic', __name__)
 
 
-@bp.route('/create', methods=['POST'])
+@bp.route('/topic/create', methods=['POST'])
 def create_topic():
-    """Handle topic creation."""
-    form = NewTopic(request.json)
-
-    if form.validate():
-        topic = Topic(name=form.name.data)
-        db.session.add(topic)
-        try:
-            db.session.commit()
-            return jsonify({"message": "Topic created successfully"}), 201
-        except Exception as e:
-            db.session.rollback()
-            return jsonify({"message": "An error occurred"}), 500
-    else:
-        return jsonify({"errors": form.errors}), 400
+    data = request.get_json()
+
+    # Validate data
+    if not data or 'title' not in data or 'parent' not in data:
```

Comment:
make a list of items you want to find in the request, and check if any item of the list not present in the request, 
for readability and easy to modify later